graylog2-server/src/main/java/org/graylog/security/shares/CollectionRequestHandler.java

```diff
@@ -19,12 +19,18 @@
 import org.graylog.grn.GRN;
 
 import java.util.Set;
+import java.util.function.Predicate;
 
 public interface CollectionRequestHandler {
     /**
-     * Pluggable handler to add given entity to the specified collections.
+     * Pluggable handler for actions related to collections.
      */
     default void addToCollection(GRN entity, Set<GRN> collections) {
         // Intentionally left empty - ignore this in the community edition.
     }
+
+    default Predicate<GRN> collectionFilter() {
```

Comment:
We don't need a default implementation here for the community edition because we don't have a class in community which implements `CollectionRequestHandler`. We can also remove the default implementation for the `addToCollection` method.

Reply:
fixed